### PR TITLE
Fixed RemoteWindowsDeployer for testing portable apps and temporary workaround for a dotnet cli issue

### PR DESCRIPTION
**Pull request by kichalla**
@JunTaoLuo 

I have tested these fixes with a Nano VM hosted locally.

### CODE REVIEW

File: src/Microsoft.AspNetCore.Server.Testing/Deployers/RemoteWindowsDeployer/StartServer.ps1 [hunk 1]
```diff
@@ -28,19 +31,28 @@ IF (-Not [string]::IsNullOrWhitespace($environmentVariables))
 	}
 }
 
+# Temporary workaround for issue https://github.com/dotnet/cli/issues/2967
+if ($executablePath -ne "dotnet.exe"){
```
Comment (kichalla) [inline src/Microsoft.AspNetCore.Server.Testing/Deployers/RemoteWindowsDeployer/StartServer.ps1]: This fix is for standalone only...for portable apps, the shared runtime needs to be fixed one time

File: src/Microsoft.AspNetCore.Server.Testing/Deployers/RemoteWindowsDeployer/StartServer.ps1 [hunk 2]
```diff
@@ -28,19 +31,28 @@ IF (-Not [string]::IsNullOrWhitespace($environmentVariables))
...
+    $destinationDir = Split-Path $executablePath
+    Copy-Item C:\Windows\System32\forwarders\shell32.dll $destinationDir
+}
+
+$command = $executablePath + " " + $executableParameters + " --server.urls " + $applicationBaseUrl
```
Comment (kichalla) [inline src/Microsoft.AspNetCore.Server.Testing/Deployers/RemoteWindowsDeployer/StartServer.ps1]: Currently the script doesn't handle paths having spaces within them. I will file a separate issue for it.

**Comment by JunTaoLuo** [comment]
:shipit:

**Comment by kichalla** [comment]
be7069b198da103410d3af8c8328d89e44a838cc

**Pull request closed without merging**